sass/color/_index.scss

```diff
@@ -102,12 +102,12 @@
 /// ```scss
 /// a:hover {
 ///   // set a background, and get well-contrasted text
-///   @include tools.contrasted('link');
-///   @include tools.contrasted(rebeccapurple);
+///   @include contrasted('link');
+///   @include contrasted(rebeccapurple);
 ///
 ///   // or return a contrasting color from a set of options
-///   border-color: tools.contrast('background', white, rebeccapurple);
-///   border-color: tools.contrast(maroon, white, rebeccapurple);
+///   border-color: contrast('background', white, rebeccapurple);
+///   border-color: contrast(maroon, white, rebeccapurple);
```

Comment:
should the `tools.` namespace be removed here in these examples?

Reply:
hmm, probably? I bet we haven't been consistent on that.

- we need the `tools` prefix on actual `@example` code, in order for it to compile
- Probably we should leave it off in the prose examples?

We can catch mistakes on that when we're doing a final review of the docs, but I think this is right.

Reply:
Good point. Yeah, Im not sure we have been consistent... but I think mostly we've used it on the `@examples` but I didn't check close on these snippets to see if we added it or not.